spec.bs

```diff
@@ -556,7 +556,7 @@ To <dfn lt="deleting the database|delete the database">delete the database</dfn>
 <dl class="domintro non-normative">
   <dt><code>{ |database|, |store|, |version| } = |storage|.{{StorageArea/backingStore}}</code>
   <dd>
-    <p>Asynchronously retrieves an an object containing all of the information necessary to manually interface with the IndexedDB backing store that underlies this storage area:
+    <p>Synchronously retrieves an object containing all of the information necessary to manually interface with the IndexedDB backing store that underlies this storage area:
```

Comment:
Thanks for this. Let's just do "Returns an object"?

Also feel free to add yourself to the acks.

Reply:
Done, thanks.